src/Components/Notifications/__tests__/util.jest.ts

```diff
@@ -1,4 +1,8 @@
-import { shouldDisplayNotification } from "Components/Notifications/util"
+import {
+  shouldDisplayNotification,
+  hasNewNotifications,
```

Comment:
Really, this can't be used like this. I'd consider just extracting all of this into a hook, and set a state flag in a useEffect.